def error(line, col, msg):
    print(line, col, msg)
    exit(1)